docker/Jenkinsfile-build-docker

```diff
@@ -1,7 +1,9 @@
-@Library('test-shared-library') _
+@Library('test-shared-library@dai-updated') _
 import ai.h2o.ci.buildsummary.StagesSummary
 
-DEFAULT_LABEL = 'mr-0xc10'
+DEFAULT_LABEL = 'linux'
```

Comment:
I expect that this is running on linux node, the rest of it is running on H2O-3 node, that is the difference there?